docs/modules/ROOT/pages/_partials/add-fuse-aggregation-app-endpoint.adoc

```diff
@@ -4,37 +4,42 @@
 
 
 [id='add-fuse-aggregation-app-endpoint_{context}']
-= Add the Fuse Aggregation App Endpoint to the {3Scale-ProductName}
+= Adding the Fuse Aggregation App Endpoint to {3Scale-ProductName}
```

Comment:
@finp 
These headings are not shown in the webapp.
I'd say that's a bug. I can create an issue if you agree

![image](https://user-images.githubusercontent.com/878251/47148535-dbef8600-d2c8-11e8-9183-b79ef6dc4982.png)